bayesian/v2/stack_analyses.py

```diff
@@ -0,0 +1,369 @@
+# Copyright © 2020 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Dharmendra G Patel <dhpatel@redhat.com>
+#
+"""Implements /api/v2/stack-analyses REST APIs for POST and GET."""
+
```

Comment:
For Path and File naming convention, Please refer https://github.com/fabric8-analytics/fabric8-analytics-server/pull/610

Reply:
Comment taken.